src/magento/magento-marketplace-synchronize-store.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Synchronize Your Store
+group: getting-started
+---
+
+{: .bs-callout-info}
+If your Magento installation is hosted on Adobe Commerce Cloud, see [Install, manage, and upgrade extensions][1].
+
+The Web Setup System Configuration tool uses the public and private keys from your Marketplace [profile]({% link magento/magento-marketplace-credentials.md %}) to synchronize your store with Magento Marketplace. After your store is synchronized, the services remain in sync so you can easily add extensions, as well as keep track of all extensions you have purchased and installed.
+
+## Step 1: Retrieve Your Marketplace Credentials
+
+1. If continuing from the previous topic, your browser is already open to the Access Key page of your Marketplace profile. Otherwise, log in to your Magento account to retrieve your Magento Marketplace access key.
+
+    If you need help, see [Set Up Your Marketplace Credentials]({% link magento/magento-marketplace-credentials.md %}).
+
+1. Find your Marketplace Access Key in the list, and **Copy** your _Public Key_. (Keep this browser tab open and continue to the next step.)
+
+## Step 2: Synchronize Your Store
+
+1. Open a new browser tab, and log in to the Admin of your store as a user with full administrator rights.
+
+1. On the Admin sidebar, go to **System** > _Tools_ > **Web Setup Wizard**.  Then, choose **System Configuration**.
```

Comment:
```suggestion
1. On the _Admin_ sidebar, go to **System** > _Tools_ > **Web Setup Wizard**.  
```

This should be the next step:

Click **System Configuration**.

Reply:
Fixed.